hamilton/driver.py

```diff
@@ -363,6 +399,15 @@ def __init__(
             # TODO -- update this to use the lifecycle methods
             self.capture_constructor_telemetry(error, modules, config, adapter)
 
+    def get_graph(self) -> graph_types.HamiltonGraph:
+        """Gives a graph object that you can use -- export, inspect, walk, etc...
+        This is in place of accessing the internal graph directly, which does not commit
+        to a stable API.
+
+        :return: A HamiltonGraph object.
+        """
+        return graph_types.HamiltonGraph.from_graph(self.graph)
```

Comment:
I don't think we shouldn't expose this quite so directly -- any export functions would utilize this logic though.